### ISSUE TITLE
Does not surpport column names complete when Alter table?

### ISSUE THREAD

**Issue by lookforit** [dbcli/pgcli#384]
For example:
```sql
ALTER TABLE tbl          
DROP COLUMN xxx 
```
I mean the xxx is column name for table tbl,but which does not auto  pop up. 
Any way to make it worked?

**Comment by amjith** [comment]
Yes, that is certainly possible. I'll take a stab.

**Comment by amjith** [comment]
This is now available in v1.0. Please upgrade. 

```
pip install -U pgcli
```
